RubberduckTests/Inspections/AssignedByValParameterMakeLocalCopyQuickFixTests_VBABlocks.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
```

Comment:
This class is worthless as a standalone class, and the members should probably be properties anyway. Why don't you just embed the input/expected content into the unit tests, which are the only place this code is even relevant?